### ISSUE TITLE
Something is not right with Travel calculations

### ISSUE THREAD

**Issue by kaushikbm** [fr3ak2402/GalaxySlicer#35]
Travel time in a print in GalaxySlicer is showing an estimate of 00:57:54 minutes where as in Creality Print is just 00:06:24 !! Following are the screenshots for comparison.

In GalaxySlicer
![image](https://github.com/fr3ak2402/GalaxySlicer/assets/5709898/2142a83d-d023-45fb-8c74-13c6a9a0e6e1)

In Creality Print
![image](https://github.com/fr3ak2402/GalaxySlicer/assets/5709898/d4fcbc0c-9861-4a8b-beb4-8c46fe87064f)

**Comment by fr3ak2402** [comment]
Is the problem still present in the beta version?

**Comment by kaushikbm** [comment]
Yes. It still has the issue. I took a model. Kept the parameters in both the slicers more or less the same. GS-beta version is showing  42:58 51.9 % and CS is showing 5:18 10.8%

**Comment by fr3ak2402** [comment]
Okay, is this just a display error or does it actually take longer to print?

**Comment by kaushikbm** [comment]
It actually takes almost the same time it says.

**Comment by kaushikbm** [comment]
@fr3ak2402 Could you find the issue ? I'm trying to setup the compiling environment in Windows. If it works maybe I'll also be able to contribute :)

**Comment by kaushikbm** [comment]
As Orca slicer released 1.8.0 beta-1 today, I sliced one of my parts in it and was surprised to see the travel calculation was much better than before.

Then I started comparing profile parameters between OS and GS, and found that there were a few big differences in Galaxy slicer profiles. I started checking and updating profile parameters.

After updating them in GS results were at par with OS. And the TRAVEL calculations are **BETTER** in GS than OS !! 👯 :D

But still CS wins the game in travel calculations :D

In total print time OS is the least, than CS and then GS.

GS screenshot
![image](https://github.com/fr3ak2402/GalaxySlicer/assets/5709898/0b3233ce-ef2e-4710-9066-4a25c19db21f)

OS screenshot
![image](https://github.com/fr3ak2402/GalaxySlicer/assets/5709898/f7c58830-12d5-4e8f-8952-d88695b2562a)

CS screenshot
![image](https://github.com/fr3ak2402/GalaxySlicer/assets/5709898/75d1507a-23aa-4ae7-8dca-fd9dcfbf01b7)
